### ISSUE TITLE
Getting access to the output of the hidden layers?

### ISSUE THREAD

**Issue by OmidSaremi** [tensorflow/ranking#91]
I am trying to figure out what my ranking model learns by visualizing the output of the hidden layers. How do I access the output of the hidden layers defined in _score_fn? Are the examples available? Thank you.

**Comment by knightBoy** [comment]
I also try to get the output of the hidden layers `in _score_fn` for prediction. Are there solutions for this problem?

**Comment by knightBoy** [comment]
I find a solution. I pass the output of the hidden layer to logits when predicting, and define my own GroupwiseRankingModel by letting the `_compute_logits_impl` function just return the scores.

**Comment by ramakumar1729** [comment]
Glad you were able to find a solution! Closing this for now.

**Comment by vkaul11** [comment]
@knightBoy how do you ensure that you are using the same checkpoint as what you use for training when you predict? Won't your change distort the result?
